src/componentManifest.ts

```diff
@@ -51,6 +51,8 @@ interface CompanionManifest extends ComponentManifest {
   settings?: {
     main: string;
   };
+  appClusters?: string[];
```

Comment:
appClustersIds / appClustersIDs ?

Reply:
The fields here aren't determined by us, but instead by the mobile/gallery teams. I can send you a link with the spec.